docs: rebranding changes (#3014)
chanzuckerberg/single-cell-data-portal: frontend/doc-site/05__Annotate and Analyze Your Data/5_4__Annotating Data.mdx

# Annotate Data

> ## Creating Annotations in CELLxGENE

Reviewer: ```suggestion
## Creating Annotations in CELLxGENE Annotate
```

### 2. Loading, editing and updating existing draft annotations

> CELLxGENE allows you to load and edit compatible draft annotations across multiple sessions.

Reviewer: ```suggestion
CELLxGENE Annotate allows you to load and edit compatible draft annotations across multiple sessions.
```

Compatible annotations are tabular, with category names as column headers; `anndata.obs.index` as the index; and categorical values (i.e., fewer unique values per column than specified in `--max-category-items`, default 1000).

There are two options for updating draft annotations.

#### Autodetect annotations CSV

> CELLxGENE will automatically find and reload your draft annotations in editable mode. This assumes that:

Reviewer: ```suggestion
CELLxGENE Annotate will automatically find and reload your draft annotations in editable mode. This assumes that:
```

1. The h5ad filename is the same
2. You launch cellxgene from the `annotations-directory` (i.e., the directory that contains your CSV)
3. You use the same browser and have not cleared your cookies (we use a small cookie to keep track of which user created the file to avoid accidental overwrites; see FAQ)

#### Specify an annotations CSV

> _This mode is only appropriate for single-user, local CELLxGENE instances._

Reviewer: ```suggestion
_This mode is only appropriate for single-user, local CELLxGENE Annotate instances._
```

As described in the [hosted tutorials](/docs/04__Analyze%20Public%20Data/4_1__Hosted%20Tutorials), we do not officially support hosted or multi-user use of cellxgene. However, we recognize that the app is often adapted for this purpose, and have tried to provide a "safe path" for multi-user setups that avoids overwriting data.

> Specifying a single file name for multiple contributors will result in data overwriting. To avoid this, you can instead specify an output directory and allow CELLxGENE to assign filenames.

Reviewer: ```suggestion
Specifying a single file name for multiple contributors will result in data overwriting. To avoid this, you can instead specify an output directory and allow CELLxGENE Annotate to assign filenames.
```

For each user, annotations will be saved as follows:
Each user will be prompted to enter a name for their annotations the first time they create a new category

> We also assign a unique identifier in the form of an 8-character suffix, `########`; this helps CELLxGENE identify their specific file to avoid overwriting others' work

Reviewer: ```suggestion
We also assign a unique identifier in the form of an 8-character suffix, `########`; this helps CELLxGENE Annotate identify their specific file to avoid overwriting others' work
```

#### What about creating continuous annotations?

> Continuous metadata is important! However, these values (e.g., pseudotime) are the result of statistical analyses that are beyond CELLxGENE's visualization and exploration focused scope. We do, of course, provide visualization of continuous metadata values computed elsewhere and stored in `anndata.obs.`

Reviewer: ```suggestion
Continuous metadata is important! However, these values (e.g., pseudotime) are the result of statistical analyses that are beyond CELLxGENE Annotate's visualization and exploration focused scope. We do, of course, provide visualization of continuous metadata values computed elsewhere and stored in `anndata.obs.`
```

#### I keep getting weird index errors when trying to join my annotations to my anndata?

This is most likely because the h5ad file you are working with is not the original file used to generate the annotations! We recommend merging new annotations in on a regular basis for this reason.

> #### How do you remember my unique ID to match my CELLxGENE session with my annotations file?

Reviewer: ```suggestion
#### How do you remember my unique ID to match my CELLxGENE Annotate session with my annotations file?
```
